.github/workflows/build.yaml

```diff
@@ -29,8 +29,7 @@ jobs:
         run: bash build.sh
       - name: Build Docker
         run: | 
-          cd aws_spark
```

Comment:
don't we still need this `cd`?

Reply:
no, since I moved the Dockerfile out of the folder